src/hooks/useValidation.js

```diff
@@ -97,75 +144,113 @@ const useValidation = (credentials, userSignup, clubSignup) => {
         error: true,
         message:
           "Club description must be between 100 and 1000 characters long",
+        field: "description",
       });
     }
 
     if (!credentials.iframe) {
       errors.push({
         error: true,
         message: "Please enter your club iframe code",
+        field: "iframe",
       });
     }
   }
 
   if (credentials.website) {
     if (!urlRegex.test(credentials.website)) {
```

Comment:
```suggestion
  if (credentials.website && !urlRegex.test(credentials.website)) {
```

Reply:
Also please fix this issue (Brave browser)
![image](https://github.com/MilanCommunity/Milan/assets/72851613/17a66b88-bb02-4c75-baf2-2e93896f952c)

Reply:
Do you mean the overlapping of text and eye icon?

Reply:
As you can see, the eye icon is not in place if the errors appear. The eye icon should be inside the box.

Reply:
The suggested edits are done, thanks